app/screens/settings/display_theme/display_theme.tsx

```diff
@@ -42,17 +42,13 @@ const DisplayTheme = ({allowedThemeKeys, componentId, currentTeamId, currentUser
             value: JSON.stringify(themeJson),
         };
         savePreference(serverUrl, [pref]);
-    }, [allowedThemeKeys, initialTheme, currentTeamId, currentUserId, serverUrl, newTheme]);
+    }, [allowedThemeKeys, currentTeamId, currentUserId, serverUrl, newTheme]);
```

Comment:
initialTheme is a ref, thus would not cause any re-updating with useCallback anyway.